docs/GetStarted/getting_started_android_cmake.md

```diff
@@ -77,7 +84,31 @@ $ cmake -G Ninja -B build-android/  \
     does [not support](https://github.com/google/iree/issues/1269) GCC well at
     the moment.
 
-Build all targets:
+### Configure on Windows
+
+On Windows, we will need the full path to the `cl.exe` compiler. This can be
+obtained by [opening a developer command prompt window](https://docs.microsoft.com/en-us/cpp/build/building-on-the-command-line?view=vs-2019#developer_command_prompt) and type
+`where cl.exe`. Copy the path and substitue all `\` with `/` to get the
+CMake-style path. Then in a command prompt (`cmd.exe`):
+
+```cmd
+REM Assuming in IREE source root
+> cmake -G Ninja -B build-android  \
+    -DCMAKE_TOOLCHAIN_FILE="%ANDROID_NDK%/build/cmake/android.toolchain.cmake" \
+    -DANDROID_ABI="arm64-v8a" -DANDROID_PLATFORM=android-29 \
+    -DIREE_BUILD_COMPILER=OFF -DIREE_BUILD_TESTS=OFF -DIREE_BUILD_SAMPLES=OFF \
+    -DIREE_HOST_C_COMPILER="<cmake-style-path-to-cl.exe>" \
+    -DIREE_HOST_CXX_COMPILER="<cmake-style-path-to-cl.exe>" \
+    -DLLVM_HOST_TRIPLE="x86_64-pc-windows-msvc" \
+    -DCROSS_TOOLCHAIN_FLAGS_NATIVE="-DCMAKE_C_COMPILER=\"<cmake-style-path-to-cl.exe>\";-DCMAKE_CXX_COMPILER=\"<cmake-style-path-to-cl.exe>\""
+```
+
+* See the Linux section in the above for explanations of the used arguments.
+* We need to define `LLVM_HOST_TRIPLE` and `CROSS_TOOLCHAIN_FLAGS_NATIVE` in the
+  above because LLVM cannot properly detect host triple under Android CMake
+  toolchain file. This might be fixed later.
+
+### Build all targets
 
 ```shell
 $ cmake --build build-android/
```

Comment:
This failed for me the first time (after fiddling with the configure flags), but succeeded after a clean rebuild.